test/cppsuite/test_harness/test_harness.hpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
```

Comment:
I'd say it makes more sense to move these includes to `poc.cxx`.

Reply:
8fe7d6e